### PR TITLE
dockstore/dockstore#2917 added accounts icons

### PR DESCRIPTION
**Pull request by emlys**
dockstore/dockstore#2917

### CODE REVIEW

**Comment by garyluu** [comment]
We have angular flexlayout directives for all flex-related styles

**Comment by agduncan94** [comment]
@emlys there is some good documentation on angular flex at https://github.com/angular/flex-layout/wiki

File: src/app/loginComponents/accounts/external/accounts.component.css [hunk 1]
```diff
@@ -12,3 +12,8 @@
 mat-card-footer {
   padding: 24px;
 }
+
+.account-logo {
```
[the remark this answers is not in the document]
Comment (emlys) [reply src/app/loginComponents/accounts/external/accounts.component.css]: Just curious, what's bad about pixels?

[the remark this answers is not in the document]
Comment (coverbeck) [reply src/app/loginComponents/accounts/external/accounts.component.css]: They don't scale well, e.g., if you are on a mobile device or high resolution monitor.

@garyluu The [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide) doesn't cover this and actually has examples with pixels. :( Maybe it's worth updating?

**Comment by emlys** [comment]
Screen Shot 2020-01-13 at 6 46 50 PM
Had to make some more changes because I realized the spacing was bad when you have an account linked. IMO the linked account version looks better like this; let me know what you think

File: src/app/loginComponents/accounts/external/accounts.component.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (garyluu) [reply src/app/loginComponents/accounts/external/accounts.component.css]: Go ahead :smile:

File: src/app/loginComponents/accounts/external/accounts.component.html [hunk 2]
```diff
@@ -15,78 +15,106 @@
...
+          color="secondary"
+          ngxClipboard
+          [cbContent]="row.source | getTokenContent: tokens"
+          (cbOnSuccess)="isCopied1 = true"
+        >
+          <mat-icon>file_copy</mat-icon>
+        </button>
+        <span *ngIf="row.show" style="overflow: auto;">{{ row.source | getTokenContent: tokens }}</span>
+        <span *ngIf="row.isLinked" class="pull-right">
+          <span class="mr-2 ml-2">
```
Comment (garyluu) [inline src/app/loginComponents/accounts/external/accounts.component.html]: mx-2

File: src/app/loginComponents/accounts/external/accounts.component.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (garyluu) [reply src/app/loginComponents/accounts/external/accounts.component.css]: https://engageinteractive.co.uk/blog/em-vs-rem-vs-px

[the remark this answers is not in the document]
Comment (coverbeck) [reply src/app/loginComponents/accounts/external/accounts.component.css]: Updated wiki with your link. I've been a heavy pixel user in the past and was looking forward to your explanation, but the link will do :)

**Pull request merged**